README.md

```diff
@@ -46,13 +45,17 @@ Next generation forum software. Epochtalk forum software utilizes technologies s
 $ git clone git@github.com:epochtalk/epochtalk.git
 ```
 
-#### 2) Change directories and install dependencies using [yarn](https://yarnpkg.com)
+### 2) SSL and Nginx setup
+
+Install an SSL cert and edit the nginx .conf file (located /etc/nginx/default.conf) with server info and SLL cert info
```

Comment:
The changes look good! Thank you for contributing, we appreciate it! If you can fix the SLL typo toward the end of the sentence we can merge your changes!